curve25519-dalek/src/backend/serial/u64/field.rs

```diff
@@ -362,6 +362,24 @@ impl FieldElement51 {
         ])
     }
 
+    /// Load a `FieldElement51` from 64 bytes, by reducing modulo q.
+    pub fn from_bytes_wide(hash: &[u8; 64]) -> FieldElement51 {
```

Comment:
Question: this seems to only be implemented for the serial backends. Do the vector backends not need something like this?

Reply:
I added them only in serial backends to follow how the library is structured. These functions `from_bytes_wide` only exist for serial in the library.